src/Parser/Hunter/Survival/Modules/Abilities.js

```diff
@@ -1,29 +1,49 @@
-import React from 'react';
-
 import SPELLS from 'common/SPELLS';
 import ITEMS from 'common/ITEMS';
 import CoreAbilities from 'Parser/Core/Modules/Abilities';
-import SpellLink from 'common/SpellLink';
-import ItemLink from 'common/ItemLink';
 
 class Abilities extends CoreAbilities {
   spellbook() {
     const combatant = this.selectedCombatant;
     return [
       {
-        spell: SPELLS.MONGOOSE_BITE_TALENT,
+        spell: SPELLS.KILL_COMMAND_SV,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        gcd: {
+          base: 1500,
+        },
+        charges: combatant.hasTalent(SPELLS.ALPHA_PREDATOR_TALENT.id) ? 2 : 1,
+        cooldown: haste => 6 / (1 + haste),
+      },
+      {
+        spell: SPELLS.RAPTOR_STRIKE,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        gcd: {
+          base: 1500,
+        },
+      },
+      {
+        spell: SPELLS.RAPTOR_STRIKE_2,
```

Comment:
Why is there 2 Raptor Strikes? Probably a good idea to add a note explaining it to make future maintenance easier.